keyboards/cannonkeys/tomak_type_a/config.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023 Andrew Kannan
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define WS2812_PIO_USE_PIO1
+#define RGBLED_NUM 100
+#define RGB_MATRIX_LED_COUNT RGBLED_NUM
```

Comment:
RGBLED_NUM isn't used anywhere else
```suggestion
#define RGB_MATRIX_LED_COUNT 100
```